### ISSUE TITLE
Learning rate is close to zero but AP is still fluctuating.

### ISSUE THREAD

**Issue by willSapgreen** [tensorflow/models#4244]
### System information
- **What is the top-level directory of the model you are using**:
Tensorflow Object Detection API

- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
No, but I adjust the 

- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
Linux Ubuntu 16.04

- **TensorFlow installed from (source or binary)**:
Binary

- **TensorFlow version (use command below)**:
1.4.1

- **Bazel version (if compiling from source)**:
X

- **CUDA/cuDNN version**:
CUDA 9.0
cuDNN 7.0

- **GPU model and memory**:
GeForce GTX 1050, 2G memory

- **Exact command to reproduce**:
Run train.py

### Describe the problem
I use SSD-InceptionV2-Coco model to train with my own dataset ( two classes )
I intentionally decay the learning rate 50% every 2000 iteration, starting from 0.04.
My understanding is the model will stop learning when learning rate is close to zero. ( as shown below )
![learning-rate-close-to-zero](https://user-images.githubusercontent.com/6188375/39938754-d1c9b83c-5508-11e8-84d6-fb3888d0399b.png), 

and the bias/weight looks stop learning too.
![bias-weight](https://user-images.githubusercontent.com/6188375/39938794-ef2aed92-5508-11e8-9862-ffc646a4b67f.jpg)
( Another question is why bias learns at beginning but weight never learns )

However, the AP is still fluctuating. My evalulation datasetset contains 202 images, and I set shuffle to False.
As you can see that there is a pattern in the AP.
So I wonder if the AP is calculated based on part of evalulation dataset, not all 202 images in my case.
![learning-rate-close-to-zero-but-ap-flustrate](https://user-images.githubusercontent.com/6188375/39938857-2696b496-5509-11e8-9e5c-5e521d499bed.jpg)

### Source code / logs
Here is the config file
[pipeline.zip](https://github.com/tensorflow/models/files/1996341/pipeline.zip)

Thank you for precious time on my question.

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
